pylint/lint/base_options.py

```diff
@@ -401,6 +401,17 @@ def _make_linter_options(linter: PyLinter) -> Options:
                 "Useful if running pylint in a server-like mode.",
             },
         ),
+        (
+            "property-classes",
+            {
+                "default": ("abc.abstractproperty",),
```

Comment:
This feels as if it should just be part of our own internal set.

Reply:
I think I feel otherwise as it gives maximum flexibility to a user. That's the end goal of config options.

Reply:
Yeah, but then we should also remove `builtins.property`... This is just `stdlib`.

Reply:
Like I said. This should be hardcoded as this is just `stdlib`.